src/backend/test/unit/auth/auth.controller.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('AuthController', () => {
      * @Input
      *   - Authorization code
      *
-     * @Output 201
+     * @Output 200
```

Comment:
I'm pretty sure that this is actually a 201 due to the fact that it creates something. The swagger docs actually show the 'callback' as returning 201 (undocumented) on use.

Reply:
good catch, i will change to 201